test/safe_pg_migrations_test.rb

```diff
@@ -156,6 +156,7 @@ def up
 
         # The not-null constraint is added.
         "SET statement_timeout TO '5s'",
+        'UPDATE "users" SET "admin"=TRUE WHERE "admin" IS NULL',
```

Comment:
From what I understand, this was always being executed it's just that the blocking activity filter logger had a bug!